backend/driver.py

```diff
@@ -282,6 +283,30 @@ def send_columns():
         print(traceback.format_exc())
         return send_traceback_error()
 
+@app.route("/api/executiontable", methods=["POST"])
+def executions_table():
+    try:
+        request_data = json.loads(request.data)
+        user_id = request_data['user_id']
+        record = getAllUserExecutionsData(user_id)
+        return send_success({"record": record})
+    except Exception:
+        print(traceback.format_exc())
+        return send_traceback_error()
+
+@app.route("/api/executionfiles", methods=["POST"])
+def executions_files():
+    try:
+        request_data = json.loads(request.data)
+        exec_id = request_data['exec_id']
+        #"ex035ddcf7bc0d15e753c850f715aa5508ecb9df75b3c6b124327bbd5d1133397f"
+        dl_results = get_presigned_url_from_exec_file("dlp-executions-bucket", exec_id, "dl_results.csv")
```

Comment:
we should make this endpoint extensible for other training endpoints? @dwu359

Reply:
other endpoints can use the get_presigned_url_from_bucket function

Reply:
so what use cases make it good to use presigned s3 urls? @dwu359

Reply:
Basically anytime we want to temporarily authorize the user to directly upload/download files from a specific folder location within a bucket in s3. It should save us from needing to send large files from the backend directly to the frontend.

Reply:
awesome. thx. super happy u got s3 presigned url to work